pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -103,14 +103,14 @@ public CommonProfile validateToken(final String token) {
         final TokenCredentials credentials = new TokenCredentials(token, "(validateToken)Method");
         try {
             validate(credentials, null);
-        } catch (final HttpAction e) {
+        } catch (final HttpAction | CredentialsException e) {
```

Comment:
I disagree: `CredentialsException` are expected exceptions should never turn into unexpected ones (`TechnicalException`). You should return a `null` profile in that case.